modules/samples/getting_started/src/org/wso2/carbon/registry/samples/populator/CategorizeArtifacts.java

```diff
@@ -116,9 +116,9 @@ public static void main(String[] args) throws Exception {
                 for (GenericArtifact artifact : restServices) {
                     if (restServicesList.contains(artifact.getQName().getLocalPart())) {
                         String lifeCycleName = artifact.getLifecycleName();
-                        if(!BUYMORE_LIFE_CYCLE.equals(lifeCycleName)) {
-                            String environment = environments[i % 5];
-                            String level = levels[i % 4];
+                        String environment = environments[i % 5];
```

Comment:
@splinter The purpose of this is to randomly assign environments to different assets. There is a list of environments and for each asset, random environment is chosen from the list and assigned. Hence this is not a meaningful constant to have. However this code segment has been replaced by later commits and does't appear in the latest version of the file.